spring/boot-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaAutoConfigurationTest.java

```diff
@@ -168,6 +177,11 @@ public AggregatedHttpResponse get() {
         public AggregatedHttpResponse getV2() {
             throw new IllegalArgumentException();
         }
+
+        @Post("/post")
+        public JsonNode post(@RequestObject JsonNode jsonNode) {
+            return objectMapper.valueToTree(jsonNode);
```

Comment:
I guess you can just `return jsonNode` and remove `objectMapper`?

Reply:
Fixed 8171329